src/views/Events/index.js

```diff
@@ -10,15 +10,49 @@ import gordonEvent from './../../services/event';
 import EventList from '../../components/EventList';
 import GordonLoader from '../../components/Loader';
 import { gordonColors } from './../../theme';
+import clsx from 'clsx';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
 
 import './event.scss';
+import { ListItemText } from '@material-ui/core';
 
 const styles = {
   searchBar: {
     margin: '0 auto',
   },
 };
 
+  const formControl = {
+    minWidth: 120,
+    maxWidth: 300,
+  };
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const campusEvents = [
+  "CL&W Credits",
+  "Admissions",
+  "Arts",
+  "Athletics",
+  "CEC",
+  "Chapel Office",
+  "Student Life"
+]
```

Comment:
I don't think you're using this static data anywhere, but this would be almost the right way to create the filters object described below.